Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionManager.cs

```diff
@@ -118,55 +107,29 @@ public partial class InteractionManager : MonoBehaviour {
       }
     }
 
-    protected INTERACTION_SCENE _scene;
     private bool _hasSceneBeenCreated = false;
-    private bool _enableGraspingLast = false;
-
-    protected ActivityManager _activityManager = new ActivityManager();
-    protected ShapeDescriptionPool _shapeDescriptionPool;
-
-    //Maps the Interaction instance handle to the behaviour
-    //A mapping only exists if a shape instance has been created
-    protected Dictionary<INTERACTION_SHAPE_INSTANCE_HANDLE, IInteractionBehaviour> _instanceHandleToBehaviour = new Dictionary<INTERACTION_SHAPE_INSTANCE_HANDLE, IInteractionBehaviour>();
 
+    public ActivityManager _activityManager = new ActivityManager();
     protected Dictionary<int, InteractionHand> _idToInteractionHand = new Dictionary<int, InteractionHand>();
     protected List<IInteractionBehaviour> _graspedBehaviours = new List<IInteractionBehaviour>();
+    protected HeuristicGrabClassifier _grabClassifier;
+    [NonSerialized]
+    public List<PhysicsUtility.SoftContact> softContacts = new List<PhysicsUtility.SoftContact>(80);
```

Comment:
I think it might be good to wrap both of these properties somehow to make it obvious that they are internal concepts and are not really for developer consumption.  Not sure the best way though while still keeping it efficient.  

Could be as simple as a getter that returns a struct containing these variables (and any other soft contact related stuff we might add in the future).  That way too we can put a single doc on the getter saying something like 'gets the mutable struct to control the next soft contact application, only use if you are 1337 haxor'